README.md

```diff
@@ -50,6 +50,16 @@ make clean
 ```bash
 make run_tests
 ```
+The output of most unit tests can be reduced by setting the environment
+variable `LOGLVL` to 6, for minimal output.
+```bash
+export LOGLVL=6
+```
+For verbose output either of the following
+```bash
+export LOGLVL=1
+unset LOGLVL
```

Comment:
If you want to do the verbose, why do you `unset LOGLVL`?

BTW, I think the syntax of Markdown for the code snippet should have the new lines before and after the snippet like this:

This is a code snippet:

```bash
code
```

A new sentence.

Reply:
It's verbose (LOGLVL=1) is the default value if the environment variable is undefined,